### ISSUE TITLE
Création d'un canal dédié à l'intelligence artificielle

### ISSUE THREAD

**Issue by Elanis** [NotANameServer/discord#166]
Etant donné l'engouement sur les IA du grand public mais aussi du milieu tech depuis un peu moins d'un an. On a vu pas mal de discussions se dérouler sur NaN à propos de Dall-e, Midjourney, ChatGPT, etc. ainsi que les technologies derrières.

Je propose de créer un canal générique en lieu et place des deux threads IA-rtistes et Chat OpenAI permettant de discuter de ces "solutions commerciales", d'autres, mais aussi d'IA en général, que ce soit leur fonctionnement, leur conception, les questionnement éthiques/philosophiques qu'ils posent, etc.

Qu'en pensez vous ?

**Comment by maximegmd** [comment]
Ça dépend du positionnement du canal, si celui ci a pour but de parler de la technologie, de la science derrière super idée. Si c'est pour créer un canal a débat pour que tout le monde donne son avis, je n'en vois pas l'utilité.

**Comment by Elanis** [comment]
Le but n'est pas non plus de créer des débats en effet.

**Comment by zazbone** [comment]
Salut,

Je pense que c'est une bonne idée, mais uniquement pour l'aspect technique, comme un vrai salon de tech;

Les salons actuels sont bien pour partager des blagues en rapport avec les IA, ça me parait essentiel de garder ces discussions fun dans ces deux salons (ou pour `#detente` quand la hype sera redescendu).

Il y a quelques personnes qui font du ml sur le serveur, et un salon dédié pourrait encourager plus de discussions. Mais pas si l'état des discussions et le même que dans les deux salons actuels.

Aussi, ça serait peut-être utile de mettre au clair les guidelines spécifique (pour les droits d'auteurs au hasard) à appliquer pour les solutions IA.

Voilà, bisous.

p.s. Pour le pt de vu "solutions commerciales" je ne sais pas ce que tu entends exactement (l'argent, c'est de la merde man)

**Comment by Elanis** [comment]
> p.s. Pour le pt de vu "solutions commerciales" je ne sais pas ce que tu entends exactement (l'argent, c'est de la merde man)

Je parlais de parler de ce qui se fait côté ChatGPT / Dall-e / Bing / etc. 😄

**Comment by zazbone** [comment]
Oui, oui, mais plus précisément, pour répondre aux questions sur comment fonctionne-leur api ?

**Comment by maximegmd** [comment]
Donc ce serait un canal pour concentrer ce qui se fait dans les deux threads plus les conversations ponctuelles plus technique dans d'autres canaux comme #science ? Où il y aurait une cohabitation de ces anciens threads pour tout ce qui est plus de l'ordre de la discussion, l'éthique, les problématiques et la nouveau canal pour la technologie, l'utilisation que ce soit des API ou le déploiement d'un modèle, l'entraînement etc ?

**Comment by Elanis** [comment]
J'avais en tête de remplacer les anciens threaads ouaip. Après, on est ici pour en discuter

**Comment by maximegmd** [comment]
J'ai peur que ça ne soit pas très intéressant pour les personnes qui font de l'IA justement, le prompt engineering, montrer les dernières news d'OpenAI c'est vraiment pas ce qui intéresse les gens qui sont dedans. 

On est un peu sur la distinctions entre le canal #jeux et le canal #gamedev pour moi, les deux sont intéressants mais pas nécessairement compatible.

Comment ça va se passer quand on aura un mix d'une conversation sur la législation a apporter pour les droits d'auteurs et en même temps une discussion plus technique par exemple l'entraînement d'un modèle avec accelerate?

Je pense qu'il serait intéressant de définir un cadre et l'orientation prioritaire du canal, est ce un canal plutôt général pour discuter de tout ce qui tourne autour de l'IA ou un canal plus technique. Où un mix des deux mais avec un tropisme pour l'un ou l'autre, ce qui pourrait impliquer une modération particulière, pour reprendre l'exemple plus haut, mettre fin a une discussion "méta" quand une discussion plus technique s'enclenche.

**Comment by zazbone** [comment]
Ça reste un salon technique, je doutes qu'il y ai souvent des discussions qui se croisent

**Comment by Elanis** [comment]
Je ne pense pas que les discussions simultanées arriveront si souvent. Et si ca arrive, rien n'empêche de faire comme sur n'importe quel autre canal: que l'une des discussions parte en thread.

**Comment by Equinox4** [comment]
hello, l'IA est un domaine en plein essor depuis quelques années et il est vrai qu'il n'existe pas actuellement de canal permettant de vrais discussions de fond sur cette branche de l'informatique. Les threads existants sont plutôt la pour que le grand public puisse discuter de ChatGPT ou de Dall-E.
A mon sens, créer un nouveau canal permettrait d'élargir le champs de discussions autour de ce sujet de manière officielle, on pourrait donc avoir des questions et des discussions sur des aspects plus techniques liés à l'IA (l'apprentissage profond, l'interprétation du langage naturel, les données d'entrainement, les techniques pour limiter les biais, la place de l'éthique, etc), il y a je pense matière à donner plein de discussions plus intéressantes que "lol il a répondu à coté".

Si un tel canal devait voir le jour, je vois deux manière de le mette en place:
- on créé ce canal en complémentarité avec les threads "Chat OpenAI" et "IA-rtistes", les threads étant réservés pour les réactions funs avec ces outils, le canal étant réservé aux discussions pour les gens plus initiés. Cela peut sembler être un bonne solution, mais je crains que l'activité ne soit suffisante dans le canal.
- on créé ce canal en remplacement des deux threads et tout type de discussion autour de l'IA pourra se retrouver dedans, on aura donc un mélange de messages funs autour de OpenIA et Dall-E et de discussions plus sérieuses sur le domaine. Je pense que ça peut se jouer, chatGPT étant sorti depuis un certain temps maintenant, le canal ne devrait pas être inondé de ce type de messages. 

Si la deuxième solution était choisis, on pourrait aussi cadrer le sujet du canal pour laisser d'avantage de place aux discussions sérieuses afin d'encourager les initiés à discuter dedans (sans toutefois interdire les messages funs).

**Comment by Julien00859** [comment]
>     on créé ce canal en remplacement des deux threads et tout type de discussion autour de l'IA pourra se retrouver dedans, on aura donc un mélange de messages funs autour de OpenIA et Dall-E et de discussions plus sérieuses sur le domaine. Je pense que ça peut se jouer, chatGPT étant sorti depuis un certain temps maintenant, le canal ne devrait pas être inondé de ce type de messages.
> Si la deuxième solution était choisis, on pourrait aussi cadrer le sujet du canal pour laisser d'avantage de place aux discussions sérieuses afin d'encourager les initiés à discuter dedans (sans toutefois interdire les messages funs).

+1 c'est comme ça que me m'imaginais ce nouveau salon

ce serait un salon où on parlerait d'ia de manière sérieux et où lorsqu'il n'y a pas de discution en cours ce serait ok de discuter de "l'ia grand publique" avec ce que ça entraine niveau société
